packages/kit/src/views/DeviceManagement/pages/DeviceGuideModal/index.tsx

```
@@ -1,18 +1,79 @@
+import ClassicHomescreenDark from '@onekeyhq/kit/assets/device_management/classic-homescreen-dark.png';
+import ClassicHomescreenLight from '@onekeyhq/kit/assets/device_management/classic-homescreen-light.png';
+import ProHomescreenDark from '@onekeyhq/kit/assets/device_management/pro-homescreen-dark.png';
+import ProHomescreenLight from '@onekeyhq/kit/assets/device_management/pro-homescreen-light.png';
+import TouchHomescreenDark from '@onekeyhq/kit/assets/device_management/touch-homescreen-dark.png';
+import TouchHomescreenLight from '@onekeyhq/kit/assets/device_management/touch-homescreen-light.png';
 import useAppNavigation from '@onekeyhq/kit/src/hooks/useAppNavigation';
 import { ETranslations } from '@onekeyhq/shared/src/locale';
 import { EOnboardingPages } from '@onekeyhq/shared/src/routes';
+import type { IAllWalletAvatarImageNames } from '@onekeyhq/shared/src/utils/avatarUtils';
 
+import { WalletAvatar } from '../../../../components/WalletAvatar';
```

ESLint

[error] Too many parent references (4 while maximum of 3 allowed)

(import-path/parent-depth)